phase1-coordinator/src/coordinator.rs

```diff
@@ -295,39 +295,39 @@ impl SystemTimeSource {
 }
 
 impl TimeSource for SystemTimeSource {
-    fn utc_now(&self) -> DateTime<Utc> {
-        Utc::now()
+    fn utc_now(&self) -> PrimitiveDateTime {
+        PrimitiveDateTime::now()
```

Comment:
This method is deprecated, how about `OffsetDateTime::now_utc`? https://time-rs.github.io/api/time/struct.OffsetDateTime.html#method.now_utc